psydac/cad/tests/test_geometry.py

```diff
@@ -55,6 +55,7 @@ def test_geometry_2d_1():
     geo_1.export('geo_1.h5')
 
 #==============================================================================
+@pytest.mark.xfail  # igakit no longer supported
```

Comment:
I think that `@pytest.mark.skip(reason='IgaKit not supported')` is better because then we do not have to run the tests.

Reply:
indeed, seems better 👍